css/style.css

```diff
@@ -16,4 +16,243 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ /* Header */
 
+ header {
+     display: flex;
+     justify-content: space-between;
+     align-items: center;
+     padding: 1rem 3rem;
+ }
+
+ .svg-logo {
+     height: 2.5rem;
+ }
+
+ @media only screen and (max-width: 768px) {
+
+     header {
+     padding: 1rem 1.2rem;
+    }
+  
+   .svg-logo {
+     height: 2rem;
+    }
+}
+
+/* navbar*/
+
+ .nav-bar {
+     width: 50%;
+ }
+
+.navigation__list {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  padding: 0;
+}
+
+.navigation__list li:hover {
+    color: rgb(255, 115, 0);
+    cursor: pointer;
+}
+
+.active {
+    font-weight: 500;
+}
+
+/* hamburger menu*/
+
+.hamburger {
+    display: none;
+}
+
+.bar {
+    display: block;
+    width: 25px;
+    height: 3px;
+    margin: 5px auto;
+    background-color: #101010;
+}
+
+@media only screen and (max-width: 768px) {
+  
+    .navigation__list{
+        display: none;
+    }
+
+    .nav-bar {
+        display: flex;
+        justify-content: end;
+    }
+
+    .hamburger {
+        display: block;
+        cursor: pointer;
+    }
+}
+
+/* hero section */
+
+.hero-intro {
+    background-image: url("../img/first-background.jpg");
+    height: 45rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;      
+}
+
+@media screen and (max-width: 768px) {
+ .hero-intro {
+    height: 35rem;
+    background-position-x: 60%;
+    background-size: cover;
+  }
+}
+
+.section-content {
+     text-align: center;
+     color: white;
+}
+
+.section-content h1 {
+    font-size: 3rem;
+    font-weight: 300;
+    margin-bottom: 0;
+}
+
+.section-content h3 {
+    font-size: 1.6rem;
+    font-weight: 200;
+    margin-top: 0.8rem;
+}
+
+button {
+    width: 140px;
+    border-radius: 4px;
+    color:white;
+    background-color: #f15d07;
+    padding: 0.8rem;
+    border: none;
+}
+
+/* icon section */
+
+.main-icons {
+    margin:6rem 5rem ;
+}
+
+.container-icon {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: flex-end;
+}
+
+@media screen and (max-width: 768px) {
+ .container-icon{
+    flex-direction: column;
+    align-items: center;
+  }
+}
+
+.main-icons h1 {
+    text-align: center;
+    font-size: 3.5rem;
+    font-weight: 300;
+}
+
+/*single icon+text container*/
+
+.karma-icon {
+    display: flex;
+    flex-direction: column;
+}
+
+.karma-icon-svg {
+    height: 10rem;   
+}
+
+.karma-icon h2 {
+    font-weight: 400;
+    font-size: 2rem;
+}
+
+/* Addional Features section*/
+
+.additonal-features {
+    display: flex;
+    flex-direction: row;
+}
+
+@media screen and (max-width: 768px) {
+ .additonal-features{
+    display: none;
+  }
+}
+
+.feature-img-png {
```

Comment:
Is there another way you could have achieved this effect without creating such a specific class that can't (really) be reused?

Reply:
I made it less specific. "feature-img".